output.proto

```diff
@@ -41,6 +41,7 @@ message SystemHealth {
     STORAGE_SHORTAGE = 2;
     SLOWDOWN = 3;
     INTERNAL_ERROR = 4;
+    OUTPUT_BUFFER_OVERFLOW = 5;
```

Comment:
Is this for the case SENSR SDK doesn't respond? If so, what component checks the status?

Reply:
@KangJH kind reminder